src/components/CommentsCards.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import { getCommentsByReviewId } from '../utils';
+
+const CommentsCards = ({ review_id}) => {
+  const [commentsList, setCommentsList] = useState([]);
+  const [commentsLoading, setCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setCommentsLoading(true)
+    getCommentsByReviewId(review_id).then((commentsFromApi) => {
+      setCommentsList(commentsFromApi);
+      setCommentsLoading(false);
+    });
+  }, []);
+
+  console.log(commentsList);
+
+  if (commentsLoading) {
+    return <h4>Comments are loading, please wait...</h4>
+  }
+
+  return (
+    <>
+      <ul>
```

Comment:
Great use of the list here! The outer <> tag may be slightly redundant as the li wraps everything so you're only returning a single block.